cmd/copyEnumeratorInit.go

```diff
@@ -460,29 +462,23 @@ func (cca *CookedCopyCmdArgs) createDstContainer(containerName string, dstWithSA
 			return err
 		}
 
-		dstURL, err := url.Parse(accountRoot)
-
+		bsc, err := common.CreateBlobServiceClient(accountRoot, dstCredInfo, nil, options)
 		if err != nil {
 			return err
 		}
+		bcc := bsc.NewContainerClient(containerName)
 
-		bsu := azblob.NewServiceURL(*dstURL, dstPipeline)
-		bcu := bsu.NewContainerURL(containerName)
-		_, err = bcu.GetProperties(ctx, azblob.LeaseAccessConditions{})
+		_, err = bcc.GetProperties(ctx, nil)
```

Comment:
We'll save a call to service if we skip this. We already check ignore ContainerAlreadyExists in after the Create call.